src/Rings/orderings.jl

```diff
@@ -1719,7 +1723,7 @@ function induce(vars::Vector{Int}, o::ProdOrdering)
 end
 
 @doc raw"""
-    induce(vars::AbstractVector{<:MPolyRingElem}, ord::ModuleOrdering)
+    induce(vars::AbstractVector{<:MPolyRingElem}, ord::MonomialOrdering)
```

Comment:
There are also some bits of #3679 in here?

Reply:
Ouch! Good catch, not sure how that happened -- hopefully managed to remove all traces now.